tests/php/static-analysis-stubs/wordpress-defines.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+// These were copied from https://github.com/szepeviktor/phpstan-wordpress.
+
+// There are no core functions to read these constants.
+define( 'ABSPATH', './' );
+define( 'WP_DEBUG', true );
+define( 'WP_DEBUG_LOG', true );
+define( 'WPMU_PLUGIN_DIR', './' );
+define( 'WP_CONTENT_DIR', 'some_string' );
+define( 'WP_CONTENT_URL', 'some_string' );
+define( 'WP_PLUGIN_DIR', 'some_string' );
+define( 'WP_PLUGIN_URL', 'some_string' );
```

Comment:
>  // There are no core functions to read these constants.

@schlessera There may be core function for these :)

Reply:
Well, I just added the defines that were missing. Would checking for core functions add any benefit?

Reply:
> Would checking for core functions add any benefit?

Actually using core functions instead of core constants adds benefit.

- future-proof
- testable
- allows hooks to alter value

Reply:
I might look into improving all of this at a later point, but I think what we have now works for the foreseeable future.